acq4/Manager.py

```diff
@@ -415,7 +415,7 @@ def loadDefinedConfig(self, name):
             if name not in self.config['configurations']:
                 raise Exception("Could not find configuration named '%s'" % name)
             cfg = self.config['configurations'].get(name, )
-        self.configure(cfg)
+        self.configure({'storageDir':cfg})
```

Comment:
I din't think this is right--shouldn't `cfg` already be a dict with `storageDir` inside?